cmd/vic-machine/common/target.go

```diff
@@ -56,6 +58,16 @@ func (t *Target) TargetFlags() []cli.Flag {
 			Value: flags.NewOptionalString(&t.Password),
 			Usage: "ESX or vCenter password",
 		},
+		cli.BoolTFlag{ // TODO: s/BoolTFlag/BoolFlag/ - should be false by default
+			Name:        "insecure",
+			Destination: &t.Insecure,
+			Usage:       "Skip verification of ESX or vCenter host certificate",
+		},
+		cli.GenericFlag{
+			Name:  "thumbprint",
```

Comment:
what does this flag mean if the value string is omitted? Should this be cli.StringFlag?